src/catch2/catch_test_macros.hpp

```diff
@@ -195,6 +198,7 @@
   #define TEST_CASE( ... )  INTERNAL_CATCH_TESTCASE_NO_REGISTRATION(INTERNAL_CATCH_UNIQUE_NAME( CATCH2_INTERNAL_TEST_ ), __VA_ARGS__)
   #define TEST_CASE_METHOD( className, ... ) INTERNAL_CATCH_TESTCASE_NO_REGISTRATION(INTERNAL_CATCH_UNIQUE_NAME( CATCH2_INTERNAL_TEST_ ))
   #define METHOD_AS_TEST_CASE( method, ... )
+  #define TEST_CASE_FIXTURE( className, ... )
```

Comment:
I just copied what TEST_CASE_METHOD does in this case. It would seem that these are purposefully unimplemented

Reply:
`TEST_CASE_METHOD` does this though

```cpp
  #define TEST_CASE_METHOD( className, ... ) INTERNAL_CATCH_TESTCASE_NO_REGISTRATION(INTERNAL_CATCH_UNIQUE_NAME( CATCH2_INTERNAL_TEST_ ))
```

Reply:
Also please add the macro to `tests/ExtraTests/X02-DisabledMacros.cpp` to check that the compilation still works when this preprocessor branch is compiled.

Reply:
Yup this was what was causing the majority of the CI failures. Fixed now!